src/web_interface/components/database_routes.py

```diff
@@ -160,7 +160,7 @@ def _build_search_query(self):
         return json.dumps(query)
 
     def _add_hash_query_to_query(self, query, value):
-        hash_types = read_list_from_config(self._config, 'file_hashes', 'hashes')
+        hash_types = parse_comma_separated_list(getattr(cfg, 'file_hashes')['hashes'])
```

Comment:
can the list conversion be done during validation?

Reply:
Not really. Since it is not defined what keys a plugin may have in its config the logic depends on the plugin.

Reply:
Maybe we could have an optional pydantic model for plugins with custom config options. I'm looking forward to getting rid of this string parsing stuff when we change the config format to TOML. BTW: do you plan to do this as part of the config changes or do we do it after this is finished?

Reply:
> BTW: do you plan to do this as part of the config changes or do we do it after this is finished?

I'd like to do this after this is finished.
The main part of work is getting rid of the config instance that was passed around everywhere.
If this is finished migration to TOML should be rather easy.

Reply:
> Maybe we could have an optional pydantic model for plugins with custom config options.

That was my thought too, but as with the toml changes I'd like to postpone them to avoid having to rebase this branch too often.